src/dev/ci_setup/setup.sh

```diff
@@ -2,116 +2,14 @@
 
 set -e
 
-dir="$(pwd)"
-cacheDir="${CACHE_DIR:-"$HOME/.kibana"}"
-
-RED='\033[0;31m'
-C_RESET='\033[0m' # Reset color
-
-###
-### Since the Jenkins logging output collector doesn't look like a TTY
-### Node/Chalk and other color libs disable their color output. But Jenkins
-### can handle color fine, so this forces https://github.com/chalk/supports-color
-### to enable color support in Chalk and other related modules.
-###
-export FORCE_COLOR=1
-
-###
-### check that we seem to be in a kibana project
-###
-if [ -f "$dir/package.json" ] && [ -f "$dir/.node-version" ]; then
-  echo "Setting up node.js and yarn in $dir"
-else
-  echo "${RED}src/dev/ci_setup/setup.sh must be run within a kibana repo${C_RESET}"
-  exit 1
-fi
-
-
-export KIBANA_DIR="$dir"
-export XPACK_DIR="$KIBANA_DIR/x-pack"
-
-parentDir="$(cd "$KIBANA_DIR/.."; pwd)"
-export PARENT_DIR="$parentDir"
-
-kbnBranch="$(jq -r .branch "$KIBANA_DIR/package.json")"
-export KIBANA_PKG_BRANCH="$kbnBranch"
+source src/dev/ci_setup/setup_env.sh true
```

Comment:
What does `true` do here?

Reply:
I split out setup_env, because I need to set up all of the environment stuff every time `sh` is called in the Jenkinsfile.

I was originally having issues with the parallel workers each trying to re-install node/yarn, so I introduced an argument to skip node/yarn installing, and just set up the env. But now that I look at the script again, I'm not sure why that was happening in the first place. Will take a second look